src/minuteman_network_sup.erl

```diff
@@ -77,7 +77,7 @@ load_rule({Table, Chain, Rule, Args}) ->
     _ ->
       lager:debug("Loading rule: ~p", [Rule1]),
       case iptables:insert(Table, Chain, Rule1) of
-        {ok, []} ->
```

Comment:
```
-spec insert(Table :: table(), Chain :: chain(),                                                                                                                                                                                                                                                                              
             Rule :: string(), Pos :: integer()) -> {ok, string()}.                                                                                                                                                                                                                                                           
insert(Table, Chain, Rule, Pos) ->                                                                                                                                                                                                                                                                                            
    T = atom_to_list(Table),                                                                                                                                                                                                                                                                                                  
    C = string:to_upper(atom_to_list(Chain)),                                                                                                                                                                                                                                                                                 
    {ok, ?CMD(?BIN ++ " -t " ++ T ++ " --insert " ++ C ++ " " ++                                                                                                                                                                                                                                                              
                                  integer_to_list(Pos) ++ " " ++ Rule)}.
```
not sure how this worked before, but it clearly returns the cmd string.  maybe the CMD macro was disabled on the release build?